### PR TITLE
Vtp field data

### PR DESCRIPTION
**Pull request by floryst**
vtkXMLPolyDataReader now parses FieldData and stores it in output polydata.

One note: field data array references are shared across all polydata pieces generated from a single vtp file.

### CODE REVIEW

**Review by jourdain — APPROVED** [review]
LGTM just glancing through the code

File: Sources/IO/XML/XMLPolyDataReader/index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import vtkXMLReader from 'vtk.js/Sources/IO/XML/XMLReader';
 import macro from 'vtk.js/Sources/macro';
 import vtkPolyData from 'vtk.js/Sources/Common/DataModel/PolyData';
+import vtkDataArray from 'vtk.js/Sources/Common/Core/DataArray';
```
Comment (finetjul) [inline Sources/IO/XML/XMLPolyDataReader/index.js:4]: sort alphabetically

Comment (floryst) [reply Sources/IO/XML/XMLPolyDataReader/index.js:4]: I'll take care of this along with the kw-web-suite update.

**Pull request merged**